docs/reference/search/rrf.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[rrf]]
 === Reciprocal rank fusion
 
-preview::[]
+preview::["This functionality is in technical preview and may be changed or removed in a future release. The syntax will likely change before GA. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."]
```

Comment:
@abdonpijpelink please add this to `sub_searches` & `rank` as a whole.

Reply:
@benwtrent Done!